docs/content/api-overview/resources/vnet.md

```diff
@@ -14,20 +14,26 @@ The Virtual Network builder (`vnet`) is used to create Azure Virtual Network ins
-| vnet           | add_tags             | Adds a set of tags to the resource                                 |
-| vnet           | add_tag              | Adds a tag to the resource                                         |
-| subnet         | name                 | Name of the subnet resource                                        |
-| subnet         | prefix               | Subnet prefix in CIDR notation (e.g. 192.168.100.0/24)             |
-| subnet         | add_delegations      | Adds one or more delegations to this subnet.                       |
-| addressSpace   | space                | When using `build_address_space` this specifies the address space. |
-| addressSpace   | subnets              | Specifies the subnets to build automatically.                      |
-| addressSpace   | build_subnet         | Specifies the name, size, and service delegations for the subnet.  |
+| Resource       | Keyword                                 | Purpose                                                              |
+| -------------- | --------------------------------------- | -------------------------------------------------------------------- |
+| vnet           | name                                    | Sets the name of the virtual network.                                |
+| vnet           | add_address_spaces                      | Adds address spaces to the virtual network.                          |
+| vnet           | add_subnets                             | Adds subnets to the virtual network.                                 |
+| vnet           | build_address_spaces                    | Automatically builds address spaces for subnet names and sizes.      |
+| vnet           | add_tags                                | Adds a set of tags to the resource                                   |
+| vnet           | add_tag                                 | Adds a tag to the resource                                           |
+| subnet         | name                                    | Name of the subnet resource                                          |
+| subnet         | prefix                                  | Subnet prefix in CIDR notation (e.g. 192.168.100.0/24)               |
+| subnet         | add_delegations                         | Adds one or more delegations to this subnet.                         |
+| subnet         | add_service_endpoints                   | Adds one or more service endpoints to this subnet.                   |
+| subnet         | associate_service_endpoint_policies     | Associates a subnet with an existing service policy.                 |
+| addressSpace   | space                                   | When using `build_address_space` this specifies the address space.   |
+| addressSpace   | subnets                                 | Specifies the subnets to build automatically.                        |
+| subnetSpec     | name | Specifies the name of the subnet to build. |
```

Comment:
Each builder should really be its own table. We use that Resource column normally (IIRC) for when a builder has keywords that apply to multiple resources.